### ISSUE TITLE
ng-content not displaying in terminal or screen 

### ISSUE THREAD

**Issue by Arlisha2019** [angular/angular-cli#15028]
🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

Please do not file questions or support requests on the GitHub issues tracker.

You can get your questions answered using other communication channels. Please see: 
https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#question

Thank you!

🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

Hello I have tried many different ways ,per the internet, to implement the ng-content. However, I have been unsuccessful. Because I am a beginner in Angular I am unsure if it a bug or not. The only reason something is displaying currently is because I actual typed it in the "panel.component.html" myself. other than that it will not display at all. 

**panel.component.html**

> <div class="panel panel-default">
    <div class="panel-heading"> 
        <ng-content select="[heading]"></ng-content>
    </div>
    <div class="panel-body"> 
        <div class="body">
         <h2>Body</h2>
         <p>Some Content here.....</p>
        </div>
    </div>
</div>

`<div class="panel panel-default">
    <div class="panel-heading"> 
        <ng-content select="[heading]"></ng-content>
    </div>
    <div class="panel-body"> 
        <div class="body">
         <h2>Body</h2>
         <p>Some Content here.....</p>
        </div>
    </div>
</div>`

**app.component.html**

 ```
<bootstrap-panel>
    <div heading>Header</div>
    <div body></div>
</bootstrap-panel>
```

[Terminal and Display](https://www.online-convert.com/downloadFile/26c13394-d653-4ff0-a328-72a136861296/d180354b-4c5e-41f3-ba3a-731725d0ae90)

**package.json**

> {
  "name": "hello-world",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.0.3",
    "@angular/common": "~8.0.3",
    "@angular/compiler": "~8.0.3",
    "@angular/core": "~8.0.3",
    "@angular/forms": "~8.0.3",
    "@angular/platform-browser": "~8.0.3",
    "@angular/platform-browser-dynamic": "~8.0.3",
    "@angular/router": "~8.0.3",
    "@fortawesome/angular-fontawesome": "^0.4.0",
    "@fortawesome/fontawesome-svg-core": "^1.2.19",
    "@fortawesome/free-regular-svg-icons": "^5.9.0",
    "@fortawesome/free-solid-svg-icons": "^5.9.0",
    "angular-font-awesome": "^3.1.2",
    "bootstrap": "^3.3.7",
    "rxjs": "~6.4.0",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.800.6",
    "@angular/cli": "~8.0.6",
    "@angular/compiler-cli": "~8.0.3",
    "@angular/language-service": "~8.0.3",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  }
}

I am fairly new to Angular. Any help would be greatly appreciated.

**Comment by alan-agius4** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).

**Comment by Arlisha2019** [comment]
HI @alan-agius4 Thank You for your prompt response. The reason I come to this site , is because I had already did my research on this issue, nothing is working for me, nothing. I even attempted to pay a mentor and they themselves could not find the issue. Again I have searched everywhere and nothing has even attempted to fix the issue. Maybe I'll call someone from Angular myself since I can't get help from here. Thank you again.
